integration_tests/binding/binding.go

```diff
@@ -0,0 +1,230 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package binding contains an Ondatra binding wrapping knebind for lemming.
+package binding
```

Comment:
I think we should add a comment explaining the purpose of this package: // Package binding wraps knebind to allow running integration tests without the need to supply kne/ondatra flags for configuration files.